servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -1036,7 +1036,7 @@ boolean shouldRetryOperation(ResourceException ex, int retryCount) {
         return retry;
     }
 
-    public void executePutPolicyVersion(ResourceContext ctx, String domainName, String policyName, String version, String fromVersion,
+    public Policy executePutPolicyVersion(ResourceContext ctx, String domainName, String policyName, String version, String fromVersion,
```

Comment:
don't we have here the same problem that we will have if we will do from the ui, a put and then a get call?

Reply:
yes, you might have the same problem depending how quickly the read request comes from the UI and how fast the replication works in Aurora. If this is within the same region, most likely with your read you'll get the updated object. If this is cross region the chance is higher. However, since we're no longer doing two separate calls but instead the request is going to be handled from the server, it will be more correct to return an updated object from DB rather than from memory.